indexer/packages/postgres/src/types/websocket-message-types.ts

```diff
@@ -101,15 +101,15 @@ export interface OrderSubaccountMessageContents {
   id: string;
   subaccountId: string;
   clientId: string;
-  clobPairId: string;
-  side: OrderSide;
-  size: string;
-  ticker: string,
-  price: string;
-  type: OrderType;
-  timeInForce: APITimeInForce;
-  postOnly: boolean;
-  reduceOnly: boolean;
+  clobPairId?: string;
```

Comment:
Making these fields optional is a breaking change, can you update the version [here](https://github.com/dydxprotocol/v4-chain/blob/main/indexer/packages/kafka/src/constants.ts#L4)? Should also check with FE teams that this doesn't break anything.